Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -18019,14 +18019,14 @@
 			);
 			inputPaths = (
 			);
-			name = "Copy extension (temporary workaround)";
+			name = "Update extensions";
```

Comment:
Could you find a more descriptive name? And is this no longer a temporary workaround?

From what I can tell, the script replaces `EXExtensionPointIdentifier` strings from the default values provided in Info.plist, but only during production builds (when DSTROOT exists and extension have been installed to it). I'm a little curious why these can't be hard-coded in Info.plist. Do they need to be different between debug and production builds?

Reply:
That's a good point. I have updated the description to be more informative. This is still a temporary workaround, which I aim to remove soon. The patch changed the install location, so the copy step of the script is no longer needed.

Thanks for reviewing!